src/plugins/intel_cpu/src/emitters/plugin/x64/jit_eltwise_emitters.cpp

```diff
@@ -1655,7 +1655,7 @@ void jit_power_static_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs,
         h->add(h->rsp, (get_max_vecs_count() + 2) * get_vec_length());
 
         // restore k registers
-        if (isa == x64::avx512_core || isa == x64::avx512_core) {
```

Comment:
@a-sidorova looks like we still need to handle different architecture for else branch? What should be the proper arch here?

Reply:
This code part means the following: firstly we push all registers to stack to save the original values, then make function call and after all we pop all values to the corresponding registers from stack. It's classical task `preamble` and `postamle`. AVX512 also provides mask registers (which are missed on sse42 and avx2) so we have to save them in the stack too.
So we don't need to handle other isa here.
Thank you!

@zhaixuejun1993 FYI: By the way Snippets team implemented the helpers for internal call [preamble](https://github.com/BHbean/openvino/blob/748a751bf3854344fdb09e74d77188ce2de7e8e7/src/plugins/intel_cpu/src/emitters/plugin/x64/jit_emitter.cpp#L217) and [postamble](https://github.com/BHbean/openvino/blob/748a751bf3854344fdb09e74d77188ce2de7e8e7/src/plugins/intel_cpu/src/emitters/plugin/x64/jit_emitter.cpp#L251). In general we can reuse them here but I'm not sure that we should do it in this PR.

Reply:
@a-sidorova Actually I just saw the condition_1 is the same as the condition_2 in the "if( condition_1 || condition_2)", so I removed one.

Reply:
Yes, I see. A few years ago this condition was `avx512_core && avx512_common`. After migration to new version of oneDNN we have `avx512_core && avx512_core`. So everything is ok